src/otx/recipe/detection/yolox_tiny.yaml

```diff
@@ -61,6 +61,13 @@ overrides:
         - class_path: otx.core.data.transform_libs.torchvision.RandomFlip
           init_args:
             prob: 0.5
+            is_numpy_to_tvtensor: true
+        - class_path: torchvision.transforms.v2.RandomVerticalFlip
+          enable: false
+        - class_path: torchvision.transforms.v2.GaussianBlur
+          enable: false
+          init_args:
+            kernel_size: 5
```

Comment:
This will result in additional numpy->tvtensor conversion.
Why don't you insert these tv2 transforms after "Pad"?